src/styles/blocks/product-catalog.scss

```diff
@@ -0,0 +1,29 @@
+.product-catalog {
+  padding-block: 50px;
```

Comment:
The `padding-block` property is not widely supported yet. It's better to use `padding-top` and `padding-bottom` instead.